browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -293,6 +294,13 @@ void RewardsDOMHandler::GetWalletPassphrase(const base::ListValue* args) {
   }
 }
 
+void RewardsDOMHandler::GetNumExcludedSites() {
+  if (rewards_service_ && web_ui()->CanCallJavascript()) {
+    int num = (int)rewards_service_->GetNumExcludedSites();
```

Comment:
Note: This is casted from to `int` because `base::Value` doesn't support `unsigned int` types for some reason